downstream/modules/builder/ref-build-args-base-image.adoc

```diff
@@ -9,14 +9,13 @@ The `build_arg_defaults` section of the definition file is a dictionary whose ke
 | Value | Description
 
 | `ANSIBLE_GALAXY_CLI_COLLECTION_OPTS`
-| * Allows the user to pass the `–pre` flag to enable the installation of pre-releases collections
-* `-c` is the equivalent of setting `verify_ssl` to `false`
+| * Allows the user to pass arbitrary arguments to the ansible-galaxy CLI during the collection installation phase. For example, the –pre flag to enable the installation of pre-release collections, or -c  to disable verification of the server's SSL certificate.
```

Comment:
(the other values aren't bulleted, probably drop this bullet?)

Reply:
Fixed.